branch_create.go

```diff
@@ -217,6 +223,8 @@ func (cmd *branchCreateCmd) Run(
 		}
 	}
 
+	branchName := cmd.Prefix + cmd.Name
```

Comment:
I went back and forth here on whether or not we should create a new variable (as I ended up doing) or just to modify `cmd.Name` like we do in the if block above. I ultimately choose this since the above if block sets cmd.Name since it has no value. In this case, we're using the now set value plus a different command argument to create the end branch name that we use.